doc/user_guide/io.rst

```diff
@@ -772,17 +772,46 @@ Currently HyperSpy has limited support for reading and saving the TIFF tags.
 However, the way that HyperSpy reads and saves the scale and the units of TIFF
 files is compatible with ImageJ/Fiji and Gatan Digital Micrograph software.
 HyperSpy can also import the scale and the units from TIFF files saved using
-FEI, Zeiss SEM and Olympus SIS software.
+FEI, Zeiss SEM, Olympus SIS, Jeol SightX and Hamamatsu HPD-TA (streak camera)
+software.
+
+Extra loading arguments
+^^^^^^^^^^^^^^^^^^^^^^^
+
+- ``force_read_resolution`` (bool, optional): Force read image resolution using
+  the x_resolution, y_resolution and resolution_unit tags of the TIFF. Beware:
+  most software don't (properly) use these tags when saving TIFF files. Default
+  is ``False``.
+- ``hamamatsu_streak_axis_type`` (str, optional): decide the type of the
+  time axis for hamamatsu streak files:
+
+  * if ``uniform``, the best-fit linear axis is used, inducing a (small)
+    linearisation error. Initialise a UniformDataAxis.
+  * if ``data``, the raw time axis parsed from the metadata is used. Initialise
+    a DataAxis.
+  * if ``functional``, the best-fit 3rd-order polynomial axis is used, avoiding
+    linearisation error. Initialise a FunctionalDataAxis.
+
+By default, ``uniform`` is used but a warning of the linearisation error is issued.
+Explicitly passing ``hamamatsu_streak_axis_type='uniform'`` suppresses the warning.
+In all cases, the original axis values are stored in the ``original_metadata`` of the
+signal object.
```

Comment:
I suppose this is visible in the docstrings of the `hs.load` function? Or is it hidden inside other sub-loading functions?

Reply:
>  Or is it hidden inside other sub-loading functions?

Exactly this. The load docstring is encyclopedic already, so I think new information would be lost inside. By default, a warning is issued specifying that information directly in the Command Line so  people won't miss it